src/components/typography/typography.component.tsx

```diff
@@ -46,214 +53,80 @@ export interface TypographyProps extends SpaceProps, ColorProps {
   display?: string;
   /** Override the list-style-type */
   listStyleType?: string;
-  /** Override the white-space type */
+  /** Override the white-space */
   whiteSpace?: string;
-  /** Override the word-wrap type */
+  /** Override the word-wrap */
   wordWrap?: string;
-  /** Override the text-overflow type */
+  /** Override the text-overflow */
   textOverflow?: string;
   /** Apply truncation */
   truncate?: boolean;
+  /** Override the color style */
+  color?: string;
+  /** Override the backgroundColor style */
+  backgroundColor?: string;
```

Comment:
**suggestion**: again, might be wrong here but unsure if we need both `bg` and `backgroundColor` props

Reply:
@tomdavies73 because we have supported both `bg` and `backgroundColor` so far, it might be wise to continue supporting both of them for now so we don't break any styling for consumers using the shorthand `bg` prop.

The  shorthand prop is based from styled system similar to `m`, `pr`, etc, so perhaps we can discuss if we want to continue using shorthands once we start working on removing styled system (FE-5531)

Reply:
@Parsium fair, might be worth having a discussion about shorthand props now and seeing what we'd like to do. As this PR is already a breaking change, it seems like the perfect opportunity to get rid of `bg` if that's the approach we take. I'm happy with either approach though 👍

Reply:
Yeah, happy to discuss when we have some time this morning